templates/layouts/configs/_footer.html

```diff
@@ -52,5 +51,10 @@
             "url": footer_urls.terms_and_conditions,
             "target": "_blank"
         }
-    })
-}}
+} %}
+
+{% if content and content.submitted_time %}
+  {% set config = config | setAttribute("footerWarning", _("Make sure you <a href='/sign-out'>leave this page</a> or close your browser if using a shared device")) %}
```

Comment:
The `href` should use a variable and the URL should use `url_for` when using our own endpoints.

Reply:
Made it a variable using `content.sign_out_url`. Thanks!